roles/lib_openshift/src/test/unit/test_oc_project.py

```diff
@@ -115,9 +115,8 @@ def test_adding_a_project(self, mock_cmd, mock_write, mock_tmpfile_copy, mock_lo
         mock_cmd.assert_has_calls([
             mock.call(['oc', 'get', 'namespace', 'operations', '-o', 'json'], None),
             mock.call(['oc', 'get', 'namespace', 'operations', '-o', 'json'], None),
-            mock.call(['oc', 'adm', 'new-project', 'operations', '--admin-role=admin',
-                       '--display-name=operations project', '--description=All things operations project',
-                       '--node-selector=ops_only=True'], None),
```

Comment:
@kwoodson the reason this is failing in Travis is because of the way that `OpenShiftCLIConfig.stringify` is handling converting `_options`, the ordering of `_options` is not guaranteed to be consistent if it is changed: `https://docs.python.org/2/library/stdtypes.html#dict.items`

You could guarantee the order of arguments provided by stringify with something like this:
```
def stringify(self, key=None):
    ''' return the options hash as cli params in a string '''
    rval = []
    config_options = self.config_options
    for name in sorted(list(config_options.keys()), key):
        data = config_options[name]
        if data['include'] \
           and (data['value'] or isinstance(data['value'], int)):
            rval.append('--%s=%s' % (name.replace('_', '-'), data['value']))
    return rval
```
I added the `key=None` param, so that child classes could pass in they key param to [sorted](https://docs.python.org/3.5/library/functions.html#sorted) if they need to ensure a particular ordering of parameters.